.github/workflows/ci-license.yaml

```diff
@@ -41,10 +41,13 @@ jobs:
       - name: checkout
         uses: actions/checkout@v1
 
-      - name: Setup Maven Version
-        run: mvn -B --version
+      # license check fails with 3.6.2 so we have to downgrade
+      - name: Set up Maven
+        uses: aahmed-se/setup-maven@v3
```

Comment:
Seems this is related to your personal account? what is this used for? Could it be changed to non-personal related?

Reply:
I can't find an equivalent github action that works, this is published here. so we should be able to take a dependency on it. It's a versioned immutable publication.

https://github.com/marketplace/actions/setup-maven-3

Reply:
hi

it seems like fork from my project
https://github.com/stCarolas/setup-maven/network/members

can we merge changes with download url and use my version?

Reply:
@aahmed-se Can you send a pull request @stCarolas repo?